libs/net/services/Actions/Managers/IngestActionManager`.cs

```diff
@@ -54,7 +55,7 @@ public IngestActionManager(IngestModel ingest, IApiService api, IIngestAction<TO
     protected override async Task<bool> PreRunAsync()
     {
         // Always fetch the latest version of the ingest and update this manager with it.
-        var ingest = await _api.GetIngestAsync(this.Ingest.Id);
+        var ingest = await this.Api.HandleRequestFailure(async () => await this.Api.GetIngestAsync(this.Ingest.Id), this.Options.ReuseIngests, this.Ingest);
```

Comment:
Default to current ingest value.  This may cause some unexpected issues if the first attempt to fetch the ingest fails.